feature-libs/asm/components/customer-list/customer-list.component.html

```diff
@@ -294,7 +294,7 @@ <h2 id="asm-customer-list-title" class="title modal-title">
         <!-- Modal Foot -->
         <div
           class="cx-dialog-sub-header modal-header cx-dialog-foot"
-          [class.tablet-mobile]="(breakpoint$ | async) !== BREAKPOINT.md"
```

Comment:
`tablet-mobile` is useless.
use `asm-mobile-pagination` instead to make sure the align direction.